[1/4] Add get_device_stats to accelerator interface (#9586)
PyTorchLightning/pytorch-lightning: CHANGELOG.md

- Added `PL_RECONCILE_PROCESS` environment variable to enable process reconciliation regardless of cluster environment settings ([#9389](https://github.com/PyTorchLightning/pytorch-lightning/pull/9389))

> - Added `get_device_stats` to Accelerator interface and implement it for GPU and TPU ([#9586](https://github.com/PyTorchLightning/pytorch-lightning/pull/9586))

Reviewer: ```suggestion
- Added `get_device_stats` to the Accelerator Interface and add its implementation for GPU and TPU ([#9586](https://github.com/PyTorchLightning/pytorch-lightning/pull/9586))
```
